plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -648,6 +652,44 @@ function MosaicMenuItem:update()
                 self.has_description = true
             end
 
+            if not self.menu.cover_info_cache then
+                self.menu.cover_info_cache = {}
+            end
+
```

Comment:
Duplicate. (You do it below below the comment, which is good as as done by ListMenu.)

Reply:
done.